README.md

```diff
@@ -14,6 +14,12 @@
 <details><summary>Click to expand</summary>
 
 - [Install](#install)
+  - [Linux](#linux)
+    - [AppImage](#appimage)
+    - [Debian/Ubuntu](#debian/ubuntu)
+    - [Arch Linux](#arch linux)
+  - [macOS](#macOS)
+  - [From Source](#from)
```

Comment:
This is automatically generated by `hallmark` and will be overwritten, currently only supports `##` level

cc @vweevers

Reply:
If we were to add a `maxDepth` option to hallmark (which is a slippery slope), and you set it to 4 here, then note that headings like [`Code Structure`](https://github.com/deltachat/deltachat-desktop#code-structure) would also be included in the TOC.

What's the reason for wanting these subsections of `Install` in the TOC? Perhaps we can somehow translate that reasoning to code.

Reply:
Alternatively, we could make `hallmark` scan + respect the existing TOC and its depth.

Reply:
> What's the reason for wanting these subsections of `Install` in the TOC?

Maybe it's enough to have `##` sections only.